src/pages/patientView/timeline/Timeline.tsx

```diff
@@ -14,22 +14,21 @@ import {PatientViewPageStore} from "../clinicalInformation/PatientViewPageStore"
 import {ClinicalEvent, ClinicalEventData} from "../../../shared/api/generated/CBioPortalAPI";
 import DownloadControls from "../../../public-lib/components/downloadControls/DownloadControls";
 import autobind from "autobind-decorator";
+import {observer} from "mobx-react";
 
 interface ITimelineProps {
     sampleManager:SampleManager;
     store:PatientViewPageStore;
     width:number;
 }
 
+@observer
 export default class Timeline extends React.Component<ITimelineProps, {}> {
 
     private currentWidth:number;
 
     shouldComponentUpdate(nextProps:ITimelineProps){
-        if (nextProps.width !== this.currentWidth) {
-            // only rerender to resize
-            this.drawTimeline(nextProps.width);
-        }
+        this.drawTimeline(nextProps.width);
```

Comment:
@inodb is this okay?

Reply:
how come you took check off?

Reply:
because I made it into an observer so its going to rerender based on changes to observables inside the sampleManager